app/views/hyrax/embargoes/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1>Manage Embargoes</h1>
```

Comment:
Something like this: https://github.com/samvera-labs/hyrax/blob/master/app/views/hyrax/admin/admin_sets/index.html.erb